.prettierrc.yml

```diff
@@ -0,0 +1,3 @@
+trailingComma: none
+singleQuote: true
+semi: false
```

Comment:
Please remove this file. This configuration is in `.eslintrc.yml`. And you can pretty-ify code with `eslint --fix`

Reply:
Yes, I get that it’s in the eslint config. That feels like a workaround because prettier didn’t support config files for a while. The reason I like it outside of eslint is because I have my editor set up to run prettier on save if it’s available locally in the project so it’s conveinent to have those preferences declared in a format that’s native to the tool. I contribute here so infrequently that it’s not a big deal to me either way. Prettier will also read config from `package.json` if it’s the extra file you don’t like.

Reply:
I want it to live in one place. I'm fine removing it from `.eslintrc.yml` if having this config file is enough. Either way though, that is an unrelated change to this PR and thus I think should be removed (and be in its own PR)

Reply:
Sounds good

Reply:
Although if living in one place is the goal then eslint is probably the place to put it since that enforces the style. Looks like neoformat supports prettier-eslint so I'll get that configured next time I'm in here.